scanner/nodeFramework.go

```diff
@@ -0,0 +1,175 @@
+package scanner
+
+import (
+	"encoding/json"
+	"io/fs"
+	"os"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
```

Comment:
Let's user `errors` in brand-new files. 

https://pkg.go.dev/errors

errors.Is and errors.As are there. fmt.Errorf will replace errors.Wrap